android-sdk/src/main/java/com/mobilecoin/lib/BlockchainClient.java

```diff
@@ -65,6 +79,13 @@ synchronized void resetCache() {
      */
     @NonNull
     synchronized ConsensusCommon.LastBlockInfoResponse getOrFetchLastBlockInfo() throws NetworkException {
+        if(lastBlockInfoTimestamp != null) {
+            Logger.e("HERE! ", lastBlockInfoTimestamp.plus(minimumFeeCacheTTL).toString());
```

Comment:
Debug logs

Reply:
Oh, whoops, I don't know how I missed that. I will fix and then merge